test/pod_performance/test-config.yaml

```diff
@@ -0,0 +1,73 @@
+# There is a total capacity for 100 pods, divided equally between 10 queues.
+# Each queue has quota for 10 pods, and may borrow up to 5 pods quota from its
+# cohort based on the borrow ratio.
+{{ $PODS_ADMITTED := 100 }}
+{{ $QUEUES := 10 }}
+{{ $COHORTS := 2 }}
+{{ $BORROW_RATIO := 0.5 }}
+
+# As each workload has 12 pods, borrowing is required for admission, as each
+# queue only has a nominal quota for 10 pods
+{{ $PODS := 1200 }}
+{{ $WORKLOADS := 100 }}
```

Comment:
Let's set the defaults so that a single pod group fits within the nominal quota of the clusterqueue.